src/file_proxy.c

```diff
@@ -1029,33 +1004,32 @@ disk_space_remaining(const char *dir)
 	}
 }
 
-bool
+off_t
 get_backup_files(const char *dir_path, as_vector *file_vec)
 {
 	uint8_t file_proxy_type = file_proxy_path_type(dir_path);
-	bool res;
+	off_t total_file_size = 0;
 	DIR* dir;
 	struct dirent* entry;
 
 	ver("Listing backup files in %s", dir_path);
 
 	switch (file_proxy_type) {
 		case FILE_PROXY_TYPE_S3:
-			res = s3_get_backup_files(dir_path, file_vec);
+			total_file_size = s3_get_backup_files(dir_path, file_vec);
 			break;
 
 		case FILE_PROXY_TYPE_LOCAL:
```

Comment:
Maybe it would be good to create another function to mirror `s3_get_backup_files` like `local_get_backup_files`

Reply:
I noticed that the local case doesn't have an abstraction. I might add this in the future.